CMSIS/NN/Tests/UnitTest/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ project(cmsis_nn_unit_tests VERSION 0.0.1)
 
 set(CMSIS_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../..")
 
+add_compile_options(-Werror
```

Comment:
What is the optimization level here?

Reply:
Nothing set so should be the default which is -O0. Let's set it to -O0 explicitly.